feat: :sparkles: Pagina Cardapio

Charlie-Pencal opened the pull request:
Criação do html e css da pagina cardapio, com as seções variedade,nutrição,petiscos e conheça

Charlie-Pencal committed:
feat: :sparkles: Pagina Cardapio

Criação do html e css da pagina cardapio, com as seções variedade,nutrição,petiscos e conheça

Vimlm commented on cardapio.html:
```diff
@@ -1,15 +1,188 @@
+    </div>
+  </section>
+
+  <section class="pratos">
```
A section a gente utiliza para elementos do site que fazem sentido sozinhas, como se fosse uma sessão do site, nesse caso que tem só a imagem solta da pra deixar ela sozinha, só com a tag img sem essa section

Vimlm commented on css/cardapio/introducao.css:
```diff
@@ -1 +1,16 @@
+
+.cardapio{
+    display: grid;
+    height: 100%;
```
Por padrão todo elemento ele sempre terá height 100%, esse é valor default deles, da pra remover esse atributo e valor aqui.

Vimlm commented on css/cardapio/pratos.css:
```diff
@@ -1 +1,4 @@
\ No newline at end of file
+/* Codigo CSS aqui */
+.pratos{
+    width: 100%;
```
O nome dessa classe aqui terá que ser mudado por conta da retira da section acima, terá que ser colocado direto na imagem.

Vimlm commented on cardapio.html:
```diff
@@ -1,15 +1,188 @@
 <head>
-  <meta charset="UTF-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta charset="UTF-8" />
```
Esse head inteiro ele está com importações que não estão funcionando das fontes, pega um das outras páginas já pronta que ele vai ficar ceritnho. (desculpa, acabei esquecendo de avisar)

Vimlm reviewed (commented):
Coloquei algumas pequenas alterações no CR, mas parabéns pelo trabalho e organização. Qualquer dúvida só falar comigo, obrigado pela ajuda.

Charlie-Pencal committed:
fix: :bug: Bug importação na Head e ajustes Página Cardápio

Arrumado as importações da Head, bem como alterado a section pratos para imagem apenas.
